models/migrations/v165.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"context"
+
+	"code.gitea.io/gitea/modules/log"
+
+	"xorm.io/xorm"
+	"xorm.io/xorm/schemas"
+)
+
+func modifyColumn(x *xorm.Engine, tableName string, col *schemas.Column) error {
```

Comment:
This shloud be moved to  `migrations.go` file as it can be reused later for other migrations